source/device.cpp

```diff
@@ -24,4 +30,14 @@ std::shared_ptr<Link> RoutingModule::next_inlink() {
     return link;
 }
 
+std::shared_ptr<Link> RoutingModule::get_destination(
```

Comment:
And here if you will decide change in .hpp
```suggestion
std::shared_ptr<Link> RoutingModule::get__link_to_destination(
```

Reply:
Done